wikipedia-dark.user.css

```
@@ -5849,7 +5861,13 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
   }
   figure[typeof~="mw:Image/Thumb"], figure[typeof~="mw:Video/Thumb"],
   figure[typeof~="mw:Audio/Thumb"], figure[typeof~="mw:Image/Frame"],
-  figure[typeof~="mw:Video/Frame"], figure[typeof~="mw:Audio/Frame"] {
+  figure[typeof~="mw:Video/Frame"], figure[typeof~="mw:Audio/Frame"],
+  figure[typeof~="mw:Image/Thumb"] > figcaption,
+  figure[typeof~="mw:Video/Thumb"] > figcaption,
+  figure[typeof~="mw:Audio/Thumb"] > figcaption,
+  figure[typeof~="mw:Image/Frame"] > figcaption,
+  figure[typeof~="mw:Video/Frame"] > figcaption,
+  figure[typeof~="mw:Audio/Frame"] > figcaption {
```

I would say needless duplication here. Wouldnt 
```suggestion
  figure[typeof*="mw"] > figcaption {
```

Instead of this lot solve the same issue?

---

Yes, you've right. I made this change a quit time ago in a separate place so I must not have noticed it. Thanks for CR.